src/run.js

```diff
@@ -3,8 +3,17 @@
 let TaskRunner = require('./modules/task-runner');
 const _ = require('lodash');
 const getTaskConfig = require('./modules/get-task-config');
+const logger = require('winston');
 
 module.exports = (taskName, taskVars, opts) => {
+  logger.cli();
+  if (opts.verbose) {
```

Comment:
The code has been merged, but personally I would have used a dictionary here to get the winston log levels. e.g.: 

const logLevels = {
  verbose: 'verbose',
 quiet: 'error'
}